bin/push/push_remind.py

```diff
@@ -1,13 +1,59 @@
+import arrow
 import json
 import logging
 import os
 import requests
+import sys
```

Comment:
oops! I guess it was still exiting if the config wasn't available, but with a different error?!